packages/period-selector-dialog/package.json

```diff
@@ -16,7 +16,7 @@
     "prebuild": "rimraf ./build/*",
     "build": "cross-env BABEL_ENV=es babel src --out-dir build --ignore spec.js && npm run build:css",
     "build:css": "cp -R ./css/* ./build/",
-    "watch": "npm run build --  --watch",
+    "watch": "npm run build --watch",
```

Comment:
Note: I believe this will break with older verisons of `npm`, so we should probably leave it unless we're sure everyone's using a new-enough version.  What was the reason for removing the argument pass-through?

Reply:
Sorry, I was unaware of explicitly required `--`, thought it was copy paste error.
Will undo.